build.py

```diff
@@ -213,7 +213,8 @@ def determine_version(self):
                 self.py_normalized_version = pkg_info_data['version']
                 self.py_normalized_version = self.py_normalized_version.replace('-alpha-', 'a')
                 self.py_normalized_version = self.py_normalized_version.replace('-beta-', 'b')
-                self.py_normalized_version = self.py_normalized_version.replace('-rc', 'rc')
```

Comment:
Hi Dave, Could you keep the line as we updated to create RC images for drivers.
self.py_normalized_version = self.py_normalized_version.replace('-rc', 'rc')